consensus/src/epoch_manager.rs

```diff
@@ -1115,7 +1121,11 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         new_epoch_state: &EpochState,
         features: &Features,
         maybe_dkg_state: anyhow::Result<DKGState>,
+        consensus_config: &OnChainConsensusConfig,
     ) -> Result<RandConfig, NoRandomnessReason> {
+        if !consensus_config.is_vtxn_enabled() {
+            return Err(NoRandomnessReason::VTxnDisabled);
+        }
```

Comment:
I'm not sure if this check is necessary.

Reply:
We should only start rand manager when vtxn is enabled? Do you see any downside?

Reply:
no downside too 😅